styles.css

```diff
@@ -0,0 +1,598 @@
+@media screen and (min-width: 1041px) {
+
+
+
+html,
+body
+main {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  width: 100%;
+  font-family: 'Dela Gothic One', cursive;
+  font-family: 'Lato', sans-serif;
+  overflow: scroll;
+
+}
+
+
+article {
+  justify-content: center;
+  display: flex;
+  flex-direction: column;
+  outline: none;
+  border: none;
+
+}
+
+button {
+  outline: none;
+  border: none;
+
+}
+
+
+header {
+  background-color:#0AF5E8;
+  height: 15vh;
+  width: 1030px;
+  display: flex;
+  outline: none;
+  border: none;
+  align-self: center;
+  align-items: center;
+  margin-right: 10px;
+}
+
+nav {
+  display: flex;
+  height: 15vh;
+  width: 100%;
+}
+
+p {
+  display: flex;
+  flex-direction: column;
+  margin: 3px;
+}
+
+section {
+  justify-content: center;
+  display: flex;
+  flex-direction: column;
+  outline: none;
+  border: none;
+
+}
+
+.div1 {
+  grid-area: 1 / 1 / 2 / 2;
+}
+
+.div2 {
+  grid-area: 1 / 2 / 2 / 3;
+
+ }
+
+.div3 {
+   grid-area: 1 / 3 / 2 / 4;
+s
+  }
+
+.div4 {
+   grid-area: 1 / 4 / 2 / 5;
+
+ }
+.div5 { grid-area: 2 / 1 / 3 / 2; }
+.div6 { grid-area: 2 / 2 / 3 / 3; }
+.div7 { grid-area: 2 / 3 / 3 / 4; }
+.div8 { grid-area: 2 / 4 / 3 / 5; }
+.div9 { grid-area: 3 / 1 / 4 / 2; }
+.div10 { grid-area: 3 / 2 / 4 / 3; }
+.div11 { grid-area: 3 / 3 / 4 / 4; }
+.div12 { grid-area: 3 / 4 / 4 / 5; }
```

Comment:
Great job getting the cards to show, but one issue I could see happening is when more cards are added. The divs for each card seem to be hardcoded to a specific place in the grid, which means when more cards are added they will not know where to go. 

You might be able to get flex wrap to work for you like it does in the mobile view!